tests/suites/cloud/tests/multicontainer/index.js

```diff
@@ -0,0 +1,202 @@
+/* Copyright 2019 balena
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+"use strict";
+const { exec } = require("mz/child_process");
+
+const waitUntilServicesRunning = async(that, uuid, services, commit, test) => {
+  await that.context.get().utils.waitUntil(async () => {
+    test.comment(`Waiting for device: ${uuid} to run services: ${services} at commit: ${commit}`);
+    let deviceServices = await that.context.get().cloud.balena.models.device.getWithServiceDetails(
+      uuid
+      );
+    let running = false
+    running = services.every((service) => {
```

Comment:
Does this "every" loop `AND` all the return values together, so if we return `true`,`true`,`false`,`true` the value of `running=false`?

Reply:
exactly that yeah: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/every